packages/terra-doc-template/src/DocTemplate.jsx

```diff
@@ -86,6 +86,12 @@ const DocTemplate = ({ packageName, readme, srcPath, examples, propsTables, ...c
     customProps.className,
   ]);
 
+  let exampleHeader;
+  if (localExamples.length > 0) {
+    const exampleHeaderText = localExamples.length > 1 ? 'Examples' : 'Example';
+    exampleHeader = <h1 className={cx('.examples-header')} >{exampleHeaderText}</h1>;
```

Comment:
nit: There's a space leftover from the original syntax on the opening h1 tag. 

className={cx('.examples-header')}` `>

This is just preference so keep what you have if you prefer it. Another way to write this could be something like:
```
const exampleHeader = examples.length === 0 ? undefined : (
  <h1 className={cx('.examples-header')}>{localExamples.length > 1 ? 'Examples' : 'Example'}</h1>
);
```

Reply:
Updated to remove the comment.

In regards how we could write it, I'm going to leave as is for now. Its how we've been writing conditional logic so I want to stay consistent.